contributors/guide/README.md

```diff
@@ -1,4 +1,9 @@
-# Kubernetes Contributor Guide
+---
+title: "Kubernetes Contributor Guide"
+weight: 1
```

Comment:
With the weights for every one of these set to 1, they'll essentially default to alphabetical order of the titles. It might be better to adjust them for a more structured order, or at least ensure the guide (`README.md`) is the top.

Reply:
Yeah, however I didn't want to do an order until we looked at the rendered result and had a discussion on how the sidebar should be ordered, so I just defaulted to 1.